### ISSUE TITLE
It's a shame of China Open Source. 

### ISSUE THREAD

**Issue by deerleo** [HumanAIGC/AnimateAnyone#53]
Fake Open Source. That's pretty disappointing!

**Comment by AngelTs** [comment]
> Fake Open Source. That's pretty disappointing!

What do you expect from communist. The capitalist is ONE ideal better, but still ...

**Comment by gigadeplex** [comment]
Guys, calm down. You're not owed anything. You didn't do the research, nor do you have the skills or resources to do it. So what's up with the entitlement? It's their research, so their company decides what to do with it. Stop crying.

**Comment by eezywu** [comment]
> Guys, calm down. You're not owed anything. You didn't do the research, nor do you have the skills or resources to do it. So what's up with the entitlement? It's their research, so their company decides what to do with it. Stop crying.
I agree with you. It's understandable that the researchers didn't open source the code in time. Maybe they are busy or face some trouble. Some of these people are being overly critical.

**Comment by ReEnMikki** [comment]
Edgy 14-year-old kids try not to bring politics into a github repo challenge (Impossible)

**Comment by cepera-ang** [comment]
> Edgy 14-year-old kids try not to bring politics into a github repo
challenge (Impossible)

And don't realize that open source ideals are much close to communism ideas
than to capitalism. What an irony.

**Comment by AngelTs** [comment]
> open source

Open source = transparency, BUT communism is the OPPOSITE of transparency (from one "old kid")

**Comment by continue-revolution** [comment]
These Chinese big tech companies normally have a long process of review before open-source is approved internally. Wait at least one more month. Alibaba's review process is at least 2 months, from what I've heard about.

AnimateDiff also has a internal review process, but they did not create any blank repository.

**Comment by vianvio** [comment]
> > open source ideals are much close to communism ideas than to capitalism
> 
> Open source = transparency, BUT communism is the OPPOSITE of transparency (from one "old kid")

yes, PRISM is the best project for transparency in the capitalism world. Just go expose all your private data to the world

**Comment by shyyyds** [comment]
Open source or not open source is the author's own right, should not be responsible

**Comment by LLLLimbo** [comment]
> > open source ideals are much close to communism ideas than to capitalism
> 
> Open source = transparency, BUT communism is the OPPOSITE of transparency (from one "old kid")

Your view on communism is spot on, but honestly, as someone from China, what I've noticed recently is that China is kind of stuck in this tug-of-war between feudalism and capitalism. Communism isn't really a thing here. Take this warehouse's company, Alibaba, for example – they're totally capitalist, no question about it.

**Comment by chrisbward** [comment]
Anyone else noticed the other dozen Alibaba "projects" appearing on social media?

**Comment by lucasjinreal** [comment]
It's just business, if some technology can make money, why make it out? Just silently make money with it.

**Comment by deerleo** [comment]
Why create a blank project before it's been officially "approved"?

**Comment by deerleo** [comment]
> > > open source ideals are much close to communism ideas than to capitalism
> > 
> > 
> > Open source = transparency, BUT communism is the OPPOSITE of transparency (from one "old kid")
> 
> In fact, China is more capitalist than you might think, maybe even more so than some capitalist countries. ~Your view on communism is spot on, but honestly, what I've noticed recently is that China is kind of stuck in this tug-of-war between feudalism and capitalism. Communism isn't really a thing here. Take this repository's company, Alibaba, for example – they're totally capitalist, no question about it.~

Don't be naive!
Jack Ma, the co-founder of Alibaba Group, had been notably absent from public view for an extended period starting from late 2020. Why? We both know the reason. Anyone who dare to against CCP, will be punished and eventually will cooperate with the CCP. Jack Ma did not believe it at first, and Ant Group failed to IPO. The alibaba group also facing a 18.2 Billion RMB punishment. 

You tell me that's a fair and capitalist market? What a joke.

**Comment by ReEnMikki** [comment]
This thread turned into a politics cesspool...

**Comment by langai1021** [comment]
Sora is a disgrace to American open source. Ha ha~

**Comment by wubowen416** [comment]
Hilarious that some people still think of China as communism. Go and visit.

**Comment by ksasso1028** [comment]
china treating AI like us treats UFOs
